airbyte-integrations/connectors/source-mixpanel/source_mixpanel/spec.json

```diff
@@ -38,6 +38,39 @@
         "description": "The default value to use if no bookmark exists for an endpoint. Default is 1 year ago.",
         "examples": ["2021-11-16"],
         "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}(T[0-9]{2}:[0-9]{2}:[0-9]{2}Z)?$"
+      },
+      "region": {
```

Comment:
I think we should do this update in a such way so existing users do not need to modify their config files and add region param.  

If region param is missing then we should use US by default.